andela/mrm_api
# #164858933 Create mutation for the office structure

#### Description
Currently, we have a new office structure model that was created in order to stay consistent with the v2 changes, this PR sets up a mutation for the new model.
#### Type of change
- Feature (Create a mutation for the office structure)

#### How Has This Been Tested?
- Pull this branch `ft-create-mutation-office-structure-164858933`
- Run the `createOfficeStructure` mutation with 
```
mutation{
  createOfficeStructure(data: [
    {structureId: "a-web-story", name: "Office 1", level: 1, parentId: 1, tag: "office", position: "1", locationId: 2},
    {structureId: "a-web-user", name: "Office 1", level: 1, parentId: 1, tag: "office", position: "2", locationId: 2},
    {structureId: "a-web-guy", name: "Office 1", level: 1, parentId: 1, tag: "office", position: "2", locationId: 2}
  ]) {
    structure{
      structureId
      name
      level
      parentId
      tag
      position
      locationId
    }
  }
}
```
- This mutation expects a `data` parameter which has the `List` data type
- You should be able to create a new office structure and get back the data


#### Checklist:



#### Pivotal Tracker
[#164858933](https://www.pivotaltracker.com/story/show/164858933)
#### Screenshots if any
<img width="1194" alt="Screen Shot 2019-04-05 at 12 41 52 PM" src="https://user-images.githubusercontent.com/42177767/55625304-42e1f800-57a0-11e9-87e9-05fb2705130f.png">
